src/main/java/edu/rpi/legup/puzzle/lightup/rules/SatisfyNumberCaseRule.java

```diff
@@ -287,4 +287,67 @@ private List<LightUpCell> getAdjacentCells(LightUpBoard board, LightUpCell cell)
         }
         return cells;
     }
+
+    /**
+     * Returns the elements necessary for the cases returned by getCases(board,puzzleElement) to be valid
+     * Overridden by case rules dependent on more than just the modified data
+     *
+     * @param board         board state at application
+     * @param puzzleElement selected puzzleElement
+     * @return List of puzzle elements (typically cells) this application of the case rule depends upon.
+     * Defaults to any element modified by any case
+     */
+    @Override
+    public List<PuzzleElement> dependentElements(Board board, PuzzleElement puzzleElement){
```

Comment:
Space between method header and opening brace.

Reply:
added